backend/api/gitops/webhook.go

```diff
@@ -615,14 +615,6 @@ func (s *Service) RegisterWebhookRoutes(g *echo.Group) {
 				return false, nil
 			}
 
-			if !strings.HasPrefix(repo.WebURL, request.WebURL) {
```

Comment:
Does anyone remember why we need this? (cause we already use the repo external id to filter)

Reply:
And in the BitBucket, the web URL we got in the CI is `http://xxxx`, but the URL we got via API is `https://` :(